aktualizr.rb

```diff
@@ -22,9 +22,9 @@ class Aktualizr < Formula
   depends_on "python3" => :build
 
   bottle do
-    root_url "https://github.com/advancedtelematic/aktualizr/releases/download/#{version}"
+    root_url "https://github.com/advancedtelematic/aktualizr/releases/download/2020.3"
```

Comment:
`version` is defined at line 7, why re-hardcoding it here?

Reply:
I just copy-paste the formula that was autogenerated by the brew bottling process. This is their philosophy that we agreed to follow.